dashboard/test/ui/features/sharepageLogo.feature

```diff
@@ -0,0 +1,47 @@
+@no_mobile
+@as_student
+Feature: Lab share page logo
+
+  Scenario: Select the logo on an applab share page while logged in and visit the homepage
+    Given I am on "http://studio.code.org/projects/applab"
+    And I wait for the page to fully load
+    Then I click selector ".project_share"
+    And I wait until element "#sharing-input" is visible
+    And I navigate to the share URL
+    And I wait for the page to fully load
+    And element "#logo-link" is visible
+    And I press "logo-link" to load a new page
+    And I wait until current URL contains "/studio.code.org/"
```

Comment:
Thanks for the feedback @aoby ! Could you take a look at the newest commits? The tests now check that clicking the image, not the link, redirects correctly and that the image has squared edges. I couldn't find a reasonable way to check that the "STUDIO" text is gone since most of the tests check for what is there, not what isn't. Do you know of a test that could be appropriate in this case or should I go ahead and write one?